components/BookingListPreview.js

```diff
@@ -1,34 +1,57 @@
 import React from 'react';
-import Link from 'next/link';
+import Link from 'next/link'
 import dayjs from 'dayjs';
+import { __, priceFormat, getIsBookingCancellable } from '../utils/helpers';
 
-import { __, priceFormat } from '../utils/helpers';
-
-const ListingListPreview = ({ booking }) => {
-  if (!booking) {
-    return null;
-  }
-
+const BookingListPreview = ({ booking }) => {
+  const bookingId = booking.get('_id');
   const start = dayjs(booking.get('start'));
   const end = dayjs(booking.get('end'));
+  const isBookingCancelable = getIsBookingCancellable(booking.get('start'), booking.get('status'))
 
   return (
-    <div className="booking-list-preview card">
-      <p>{ __('bookings_status') } <b>{booking.get('status')}</b></p>
-      <p>{ __('bookings_checkin') } <b>{start.format('LLL')}</b></p>
-      <p>{ __('bookings_checkout') } <b>{end.format('LLL')}</b></p>
-      <p>{ __('bookings_total') }
-        <b className={ booking.get('volunteer') ? 'line-through': '' }>
-          {' '}{priceFormat(booking.get('price'))}
+    <div className="booking-list-preview card mb-4 last:mb-0 md:w-fit">
```

Comment:
Let's not use w-fit - we want rather the size to be defined by the container.

Reply:
ok, lets reset it for now and in the next pr - My bookings I will fix it with the card itself @closerearth